scripts/testedFeatures.json

```diff
@@ -11,7 +11,7 @@
   "354-creator_opens_my_publications",
   "355-creator_opens_an_item_in_my_publication_list",
   "385-creator_begins_registration_by_uploading_a_file",
-  "406-user_sees_a_subunit_from_my_profile",
+  "405-user_sees_a_subunit_from_my_profile",
```

Comment:
https://unit.atlassian.net/browse/NP-405 -> "Create own authority in ARP" 

Skal det ikke være https://unit.atlassian.net/browse/NP-407 -> "User selects a Subunit from My Profile"

Reply:
Vi valgte å navne om 406 til 405 tidligere siden vi ikke klarte å finne den som du akkurat nå fant.

Reply:
så det skal egentlig være 407?